Controller/Admin/OrderTabDetails.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use \OxidEsales\Eshop\Application\Controller\Admin\AdminDetailsController;
+use \OxidEsales\Eshop\Application\Model\Order;
+
+use \Wirecard\Oxid\Core\ResponseMapper;
+use \Wirecard\Oxid\Model\Transaction;
+
+/**
+ * Controls the view for the order details tab.
+ */
+class OrderTabDetails extends AdminDetailsController
+{
+    const NOTHING_SELECTED = '-1';
+
+    /**
+     * @inheritdoc
+     */
+    protected $_sThisTemplate = 'list_tab.tpl';
+
+    /**
+     * @var Order
+     */
+    protected $oOrder;
+
+    /**
+     * @var Transaction
+     */
+    protected $oTransaction;
+
+    /**
+     * @var ResponseMapper
+     */
+    protected $oResponseMapper;
+
+    /**
+     * OrderTabDetails constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->oOrder = oxNew(Order::class);
+        $this->oTransaction = oxNew(Transaction::class);
+
+        $sOrderId = $this->getEditObjectId();
+
+        if (isset($sOrderId) && $sOrderId !== self::NOTHING_SELECTED) {
+            $this->oOrder->load($sOrderId);
+
+            if ($this->oTransaction->loadWithTransactionId($this->oOrder->oxorder__wdoxidee_transactionid->value)) {
+                $this->oResponseMapper = new ResponseMapper($this->oTransaction->getResponseXML());
+            }
+        }
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return string
+     */
+    public function render(): string
+    {
+        $this->_aViewData += [
+            'listData' => $this->oOrder->getId() && $this->oResponseMapper
+                ? $this->getListData() : [],
+            'controller' => $this->classKey,
+        ];
+
+        return parent::render();
+    }
+
+    /**
+     * Returns an array of transaction data, sorted in a specific order, used to populate the view.
+     *
+     * @return array
+     */
+    public function getListData(): array
+    {
+        $aData = $this->oResponseMapper->getData();
+
+        if (!$aData) {
+            return array();
+        }
+
+        $aSortItems = [
+            'payment-methods.0.name',
+            'order-number',
+            'request-id',
+            'transaction-id',
+            'transaction-state',
+            'statuses.0.provider-transaction-id'
+        ];
+
+        $aRestOfKeys = array_diff(array_keys($aData), $aSortItems);
+        $aSortedKeys = array_merge($aSortItems, $aRestOfKeys);
+
+        $aList = array();
+        foreach ($aSortedKeys as $sKey) {
+            $aList[] = [
+                'title' => $sKey,
+                'value' => $aData[$sKey]
```

Comment:
You might want to check if `$sKey` exists in `$aData` (not all keys from `$aSortItems` have to be present in the response).

```suggestion
                'value' => $aData[$sKey] ?? null,
```